patches/chrome-android-java-res-layout-toolbar_phone.xml.patch

```diff
@@ -1,7 +1,25 @@
 diff --git a/chrome/android/java/res/layout/toolbar_phone.xml b/chrome/android/java/res/layout/toolbar_phone.xml
-index fe66549d3ac660c12186e0c9c78c7ed525f2015d..9f2fad8590812ececbf1b9d04ec493c2ed7da425 100644
+index fe66549d3ac660c12186e0c9c78c7ed525f2015d..edcb5a1c78ac8f3165b296af8e39b69c6961601a 100644
 --- a/chrome/android/java/res/layout/toolbar_phone.xml
 +++ b/chrome/android/java/res/layout/toolbar_phone.xml
+@@ -12,7 +12,7 @@
+     android:id="@+id/toolbar"
+     android:layout_width="match_parent"
+     android:layout_height="@dimen/toolbar_height_no_shadow" >
+-
```

Comment:
please leave existing whitespace intact to minimize the diff

Reply:
done